frontend/src/metabase/collections/context.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext, useContext, type ContextType } from "react";
+import type { DragDropContextProvider } from "react-dnd";
+
+export const DragDropContextProviderContext =
+  createContext<ContextType<typeof DragDropContextProvider>>(null);
```

Comment:
If we don't need the underlying component, can this be a boolean instead (or a boolean in a Redux store)

Reply:
Yeah that makes sense. I'll add `isDndAvailable` that gets updated in `app.js` but not other places